public/app/plugins/panel/geomap/style/types.ts

```diff
@@ -1,4 +1,5 @@
-import { Style } from 'ol/style';
+import Photo from 'ol-ext/style/Photo';
```

Comment:
I don't think we want photo specific things at this level -- can they be contained entirely in the photoLayer?